L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducer.cc

```diff
@@ -196,147 +191,6 @@ void L1MetPfProducer::CalcMlMet(const std::vector<float>& pt,
-    phiQ1 = 0;
-  }
-  if (isX) {
-    typedef ap_ufixed<14, 12, AP_RND, AP_WRAP> pt_t;  // LSB is 0.25 and max is 4 TeV
-    ap_ufixed<pt_t::width, 0> cosPhi = cos(phiQ1.to_double() / hwPiOverTwo_.to_double() * M_PI / 2);
-    pxy = pt * cosPhi;
-    if (phi > hwPiOverTwo_ || phi < -hwPiOverTwo_)
-      pxy = -pxy;
-  } else {
-    ap_ufixed<pt_t::width, 0> sinPhi = sin(phiQ1.to_double() / hwPiOverTwo_.to_double() * M_PI / 2);
-    pxy = pt * sinPhi;
-    if (phi < 0)
-      pxy = -pxy;
-  }
-}
-
-void L1MetPfProducer::PhiFromXY(pxy_t px, pxy_t py, phi_t& phi, bool debug) const {
-  if (px == 0 && py == 0) {
-    phi = 0;
-    return;
-  }
-  if (px == 0) {
-    phi = py > 0 ? hwPiOverTwo_ : phi_t(-hwPiOverTwo_);
-    return;
-  }
-  if (py == 0) {
-    phi = px > 0 ? phi_t(0) : phi_t(-hwPi_);
-    return;
-  }
-
-  // get q1 coordinates
-  pt_t x = px > 0 ? pt_t(px) : pt_t(-px);  //px>=0 ? px : -px;
-  pt_t y = py > 0 ? pt_t(py) : pt_t(-py);  //px>=0 ? px : -px;
-  // transform so a<b
-  pt_t a = x < y ? x : y;
-  pt_t b = x < y ? y : x;
-
-  if (b.to_double() > maxPt_ / dropFactor_)
-    b = maxPt_ / dropFactor_;
-  // map [0,max/4) to inv table size
-  int index = round((b.to_double() / (maxPt_ / dropFactor_)) * invTableSize_);
-  float bcheck = (float(index) / invTableSize_) * (maxPt_ / dropFactor_);
-  inv_t inv_b = 1. / ((float(index) / invTableSize_) * (maxPt_ / dropFactor_));
-
-  inv_t a_over_b = a * inv_b;
-
-  if (debug) {
-    printf("  a, b = %f, %f;   index, inv = %d, %f; ratio = %f \n",
-           a.to_double(),
-           b.to_double(),
-           index,
-           inv_b.to_double(),
-           a_over_b.to_double());
-    printf("    bcheck, 1/bc = %f, %f  -- %d  %f  %d  \n", bcheck, 1. / bcheck, invTableSize_, maxPt_, dropFactor_);
-  }
-
-  int atanTableBits_ = 7;
-  int atanTableSize_ = (1 << atanTableBits_);
-  index = round(a_over_b.to_double() * atanTableSize_);
-  phi = atan(float(index) / atanTableSize_) / phiLSB_;
-
-  if (debug) {
-    printf("    atan index, phi = %d, %f (%f rad)  real atan(a/b)= %f  \n",
-           index,
-           phi.to_double(),
-           phi.to_double() * (M_PI / hwPi_.to_double()),
-           atan(a.to_double() / b.to_double()));
-  }
-
-  // rotate from (0,pi/4) to full quad1
-  if (y > x)
-    phi = hwPiOverTwo_ - phi;  //phi = pi/2 - phi
-  // other quadrants
-  if (px < 0 && py > 0)
-    phi = hwPi_ - phi;  // Q2 phi = pi - phi
-  if (px > 0 && py < 0)
-    phi = -phi;  // Q4 phi = -phi
-  if (px < 0 && py < 0)
-    phi = -(hwPi_ - phi);  // Q3 composition of both
-
-  if (debug) {
-    printf("    phi hw, float, real = %f, %f    (%f rad from x,y = %f, %f) \n",
-           phi.to_double(),
-           phi.to_double() * (M_PI / hwPi_.to_double()),
-           atan2(py.to_double(), px.to_double()),
-           px.to_double(),
-           py.to_double());
-  }
-}
-
 L1MetPfProducer::~L1MetPfProducer() {}
 
 #include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
Remove this as it is included at https://github.com/cms-sw/cmssw/blob/88d6bf5ea35b98871e09d3f72abf0ac0acba3a41/L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducer.cc#L9